language/de-DE/de-DE.ini

```
@@ -168,6 +169,7 @@ JFIELD_TITLE_DESC="Beitragstitel"
 
 JGLOBAL_ADD_CUSTOM_CATEGORY="Neue Kategorie hinzufügen"
 JGLOBAL_ARTICLES="Beiträge"
+JGLOBAL_CONFIRM_DELETE="Sicher, dass gelöscht werden soll? Durch die Bestätigung werden die ausgewählten Artikel dauerhaft gelöscht!"
```

Hier wurde erstmalig Artikel verwendet!

Unsere Idee war ja, dass wir mit 4.0 die Chance haben, alle **Beiträge** in **Artikel** zu ändern.
Sollen wir das machen?

---

hmm hab ich auch mit @coolwebcreations angesprochen. Das bedeutet auch alle Dokumentationen, FAQ's, Büche etc nach zu ziehen bin mir nicht sicher ob es das wert ist.

---

Bei diesem speziellen String würde ich aber etwas anderes schreiben da es der Globale String für alle confirm delete abfragen ist:

```suggestion
JGLOBAL_CONFIRM_DELETE="Sicher, dass gelöscht werden soll? Durch die Bestätigung werden die ausgewählten Einträge dauerhaft gelöscht!"
```